Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:lts-alpine as builder
+
+# Install dependencies
+RUN apk add --no-cache git
+
+# Initialize the working directory
+WORKDIR /app
+
+# Copy package.json
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the source code
+COPY . .
+
+# Prune to production dependencies
+RUN npm prune --production
+
+
+# Final image
+FROM node:lts-alpine
```

Comment:
Why do you need a second image?

Reply:
This is a mutli-stage Dockerfile that lowers the final image size from `897MB` to `151MB`

Reply:
sure thing